estuary/utils/story.py

```diff
@@ -195,9 +205,157 @@ def get_wait_times(self, results):
 
             # Find the time between when the current artifact completes and the next one starts
             wait_time = next_artifact_start_time - completion_time
-            wait_times[index] = wait_time.seconds
+            wait_times[index] = wait_time.total_seconds()
+
+            # The 'wait time' between a FreshmakerEvent and a ContainerKojiBuild is still a part of
+            # the processing in a FreshmakerEvent, so we do not count it towards the total wait time
+            if artifact.__label__ != 'FreshmakerEvent':
+                total_wait_time += wait_time.total_seconds()
 
-        return wait_times
+        return wait_times, total_wait_time
+
+    def get_total_processing_time(self, results):
+        """
+        Get the total time spent processing the story.
+
+        :param list results: contains inflated results from Neo4j
+        :return: the seconds of total time spent processing
+        :rtype: int
+        """
+        total = 0
+        build = None
```

Comment:
Could you add a comment as to what this variable is for please? It just makes the code easier to read.